cmd/cosign/cli/verify/verify_blob.go

```diff
@@ -88,14 +68,20 @@ type VerifyBlobCmd struct {
 // nolint
 func (c *VerifyBlobCmd) Exec(ctx context.Context, blobRef string) error {
 	var cert *x509.Certificate
-	var bundle *bundle.RekorBundle
+	var chain []*x509.Certificate
+	opts := make([]static.Option, 0)
 
 	// Require a certificate/key OR a local bundle file that has the cert.
+	if options.NOf(c.KeyRef, c.CertRef, c.Sk, c.BundlePath) == 0 {
```

Comment:
Should this be `!= 1`? 

Oh, I guess not because `BundlePath` isn't really a full "bundle" yet. That's annoying.